dart/test/monster_test_my_game.example2_generated.dart

```diff
@@ -26,6 +27,37 @@ class Monster {
   String toString() {
     return 'Monster{}';
   }
+
+  MonsterT unPack() => MonsterT();
+
+  static int pack(fb.Builder fbBuilder, MonsterT object) {
+    if (object == null) return 0;
+    return object.pack(fbBuilder);
+  }
+}
+
+class MonsterT {
+
+  int pack(fb.Builder fbBuilder) => _pack(fbBuilder, HashMap<Object, int>());
+
+  /// Write the object to the [Builder] only once and reuse the same offset on
+  /// subsequent references of the same object.
+  ///
+  /// Note that this method assumes you call it using the same [Builder]
+  /// instance every time. The returned offset is only good for the [Builder]
+  /// used in the first call to this method.
+  int packOnce(fb.Builder fbBuilder, Map<Object, int> existingOffsets) {
+    assert(existingOffsets != null);
+    return existingOffsets[this] ??= _pack(fbBuilder, existingOffsets);
+  }
+
+  int _pack(fb.Builder fbBuilder, Map<Object, int> existingOffsets) {
+    assert(existingOffsets != null);
+    assert(fbBuilder != null);
+
+    fbBuilder.startTable();
+    return fbBuilder.endTable();
```

Comment:
Why does this just create an empty table?

Reply:
There are no properties. Yet again, see existing `ObjectBuilder.finish()`